libsql-sqlite3/src/pager.c

```diff
@@ -18,6 +18,8 @@
 ** file simultaneously, or one process from reading the database while
 ** another is writing.
 */
+#include <stdio.h>
+#include <string.h>
```

Comment:
nope and nope :innocent: whatever you need from stdio.h or string.h, there must be an SQLite internal equivalent. Or were they just debug residue?

Reply:
yes ik ik, stupic clangd keeps adding those